<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDlg</class>
 <widget class="QDialog" name="CSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>505</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="wnd_nav_btns" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_title" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_spacer">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_general" native="true"/>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_controls" native="true"/>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_keyboard" native="true"/>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_plugins" native="true"/>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_file_type" native="true"/>
        </item>
        <item>
         <widget class="CSettingsDlgNavBtn" name="pushButton_explorers" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSettingsDlgNavBtn</class>
   <extends>QWidget</extends>
   <header>csettingsdlgnavbtn.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
